docs/source/reference/command_line.rst

```diff
@@ -367,17 +367,17 @@ Below is a list with all available subcommands.
       --help  Show this message and exit.
 
     Commands:
-      call-root  Show root process of the call stack for the given processes.
+      call-root  Show root process(es) of the call stack for the given process(es).
       dump       Dump process input and output files to disk.
-      kill       Kill running processes.
+      kill       Kill the given process(es).
```

Comment:
I can see why you changed this. When looking at the help string of the actual command, this phrasing might make more sense. But in this overview, I feel it doesn't make much sense, because there are no actual process(es) given yet. The more abstract phrasing "{Action} processes" feels like a more natural description.

I think we could have both by reverting the first line of the docstring to what we had, but then add a line to the docstring that has your phrasing. Example:
```
def kill_processes(processes):
    """Kill running processes.

    Kill the running processes given by PROCESSES.
    """
```
In this way, we also include the metavar `PROCESSES` that references the automatically formatted usage line.

Reply:
Are you okay with singular/plural consideration?
```
def kill_processes(processes):
    """Kill one or multiple running processes.

    Kill the running process(es) given by PROCESSES.
    """
```
Tried to be consistent with help messages of the other commands `show`, `status` and `report`

Reply:
> Are you okay with singular/plural consideration?

I am fine with specifying that it can be one or many, I just don't think it should be in the short help text. The way click works, with the first line of the docstring being used as the short help for the command group overview, and the rest of the docstring being shown for the extended help text of the command, I think it makes sense to keep the short help short and sweet. It doesn't really matter at that point what the specific of the implementation are. Sure you can use the explicit `short_help` argument in the `command` declaration as you are doing, but this just seems like unnecessary work and duplication.

What I think I would want to see as a user when I do `verdi process --help` is
```
      call-root  Show the root process of processes.
      dump       Dump input and output files of processes to disk.
      kill       Kill running processes.
      list       Show a list of processes.
      pause      Pause running processes.
      play       Play (unpause) paused processes.
      repair     Automatically repair all stuck processes.
      report     Show the log report of processes.
      show       Show details of processes.
      status     Show the status of processes.
      watch      Watch state transitions of processes.
```

In the extended help text (the docstring body) you can then use more elaborate and specific language such as your suggestion:
```
Kill the running process(es) given by PROCESSES.
```

This gives a short and clear overview of _what_ commands do in the group help, and then in the command help text you provide details about the functionality and implementation.

What do you think?

Reply:
implemented like in your suggestion